packages/kit/types/index.d.ts

```diff
@@ -972,7 +972,7 @@ declare module '@sveltejs/kit' {
 		 */
 		route: {
 			/**
-			 * The ID of the current route - e.g. for `src/routes/blog/[slug]`, it would be `/blog/[slug]`
+			 * The ID of the current route - e.g. for `src/routes/blog/[slug]`, it would be `/blog/[slug]`. It is null when the router fails to match a route.
```

Comment:
I think we can tweak this to be a bit more concise and avoid the word "fails" which might indicate some error occurred rather than expected behavior
```suggestion
			 * The ID of the current route - e.g. for `src/routes/blog/[slug]`, it would be `/blog/[slug]`. The ID is `null` when no route is matched.
```